### PR TITLE
remove pipe error handling

### LINKED ISSUE #6: Error handling prevents true exception from being caught
Since pipe-ruby combines the ReducerError with the actual exception being raise, it prevents the error from being rescued as its not a known type. This comes into play particularly with [Sinatra's error handling](http://www.sinatrarb.com/intro.html#Error), when you can have an `error` for a custom error. If the actual error is `Namespace::NotFound` and you have a custom error handler in sinatra, it will not be executed because the error Sinatra gets is `Pipe::ReducerError::NotFound`.  Alternatively, you cannot have a custom error for `Pipe::ReducerError::NotFound` because it doesn't exist until the error is raised, causing Sinatra to complain when starting up.

### PR DESCRIPTION
**Pull request by phanyzewski**
Fixes #6

### CODE REVIEW

**Comment by dmccraw** [comment]
Just curious the reason for deleting error handling?

**Comment by phanyzewski** [comment]
Error handling in pipe-ruby does not work as intended.  See https://github.com/teamsnap/pipe-ruby/issues/6 for the full detail of the issue.

**Comment by ryanwi** [comment]
Have you used the error handlers explicitly @dmccraw? For the most part they seem to get in the way and I haven't found that we're using the handlers much. So we usually end up with hard to parse errors and/or duplicate errors in error tracking.

**Comment by dmccraw** [comment]
I haven't ever used the error handling and I don't have any problem removing them. I was just curious why the change and there weren't any details in the description so I was just asking.

**Review by ryanwi — DISMISSED** [review]

**Comment by ryanwi** [comment]
Can a changelog be added?

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Change Log
+
+## [Unreleased]
+- remove pip-ruby error handling
```
Comment (ryanwi) [inline CHANGELOG.md]: spelling and should this be noted as a 1.0.0 version?

**Comment by phanyzewski** [comment]
https://rubygems.org/gems/pipe-ruby/versions/1.0.0

**Review by ryanwi — APPROVED** [review]
